cosmos/lib/cosmos/api/target_api.rb

```diff
@@ -17,6 +17,8 @@
 # enterprise edition license of COSMOS if purchased from the
 # copyright holder
 
+
+autoload(:Aws, 'cosmos/utilities/s3_autoload.rb')
```

Comment:
This should be require 'cosmos/utilities/s3' not autoload

Reply:
I didn't really want that code so I looked at how s3 was loading and copied that.